uniffi_macros/src/util.rs

```diff
@@ -276,3 +276,20 @@ impl Parse for ExternalTypeItem {
         })
     }
 }
+
+pub(crate) fn extract_docstring(attrs: &[Attribute]) -> syn::Result<String> {
```

Comment:
also space before "foo", but yeah, we should. IMO we should also check udl does the same?

Reply:
to speed this along I'm gonna merge and tackle that separately

Reply:
> Turns out: not a problem in generated code because of the use of textwrap::dedent (and textwrap::indent)

However, useless spaces would be included in the metadata in library files, right?
So maybe adding a trim here would be a good thing:
https://github.com/mozilla/uniffi-rs/blob/main/uniffi_macros/src/util.rs#L288

Do you want me to do a mini-PR?

Reply:
True. I'll gladly accept a PR

Reply:
@badboy https://github.com/mozilla/uniffi-rs/pull/1878